pyte/screens.py

```diff
@@ -450,7 +450,8 @@ def draw(self, data):
 
             line = self.buffer[self.cursor.y]
             if char_width == 1:
-                line[self.cursor.x] = self.cursor.attrs._replace(data=char)
+                if char != self.cursor.attrs.data:
```

Comment:
I think you should be comparing to default char there, rather than the cursor, right?

Also, commits like that should be complemented by a short motivation statement and (optionally) some measurements.